packages/cfd/src/Stores/Modules/CFD/cfd-store.js

```diff
@@ -126,9 +123,14 @@ export default class CFDStore extends BaseStore {
             : '';
     }
 
+    get has_submitted_cfd_personal_details() {
+        const { citizen, place_of_birth, tax_residence, tax_identification_number, account_opening_reason } =
+            this.root_store.client.account_settings;
```

Comment:
does this need optional chaining?

Reply:
I need to check whether all fields are not null